src/Command/CommandInterface.php

```diff
@@ -833,4 +833,14 @@ public function upsert(
         bool|array $updateColumns = true,
         array $params = []
     ): static;
+
+    /**
+     *
+     * Execute `REFRESH MATERIALIZED VIEW` command
+     * @param string $viewName
+     * @param bool|null $concurrently If `null` then auto choice from depends on DBMS
+     * @param bool|null $withData
+     * @return bool
+     */
+    public function refreshMaterializedView(string $viewName, ?bool $concurrently = null, ?bool $withData = null): bool;
```

Comment:
```suggestion
    public function refreshMaterializedView(string $viewName, bool $concurrently = false, ?bool $withData = null): bool;
```

Why the interface is different from `DDLQueryBuilderInterface::refreshMaterializedView()`?
Looks like should be the same.

When execute `$command->refreshMaterializedView($viewName);` expect this is not concurently.

Reply:
From what I see in [MS SQL](https://learn.microsoft.com/ru-ru/sql/t-sql/statements/alter-materialized-view-transact-sql?view=azure-sqldw-latest) / [Oracle](https://docs.oracle.com/en/database/oracle/oracle-database/18/dwhsg/refreshing-materialized-views.html#GUID-EB813C4E-FEE4-4E4C-AAB8-9A34F8EB8D3C) / [Clickhouse](https://clickhouse.com/docs/en/guides/developer/cascading-materialized-views#combining-multiple-source-tables-to-single-target-table) docs:
1. `WITH [NO] DATA` is PostgreSQL only option and pretty much useless one.
2. `REFRESH` exists only on PG and Oracle. MS SQL and Clickhouse do automatic updates, so their implementations should always return `true`.
There are a lot of variants in [Oracle](https://docs.oracle.com/en/database/oracle/oracle-database/18/dwhsg/refreshing-materialized-views.html#GUID-51191C38-D52F-4A4D-B6FF-E631965AD69A): 3 commands (`REFRESH`, `REFREASH_ALL_MVIEWS`, `REFRESH_DEPENDENT`) and at least 6 options. Option `out_of_place` seems to be equivalent of CONCURRENTLY in PostgreSQL.

So, the most common part is refresh + option for non-blocking update, but it will be very hard to take full advantage  of database capabilities.